packages/workflow/src/records/handler/create.ts

```diff
@@ -290,7 +290,6 @@ export default async function createRecordHandler(
   const notificationDisabled =
     isHospitalNotification(record) ||
     event === EVENT_TYPE.MARRIAGE ||
```

Comment:
@Nil20 Could remove this marriage event check please? This is not needed anymore as the `isNotificationEnabled` function handles the marriage event now